examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -330,40 +366,42 @@ def run_validation_edges(validation: Dataset, params, systems, epoch):
     num_epochs = int(np.ceil(args.param_updates / len(relative_transformations)))
     np.random.seed(args.seed)
 
+    batch_size = args.batch_size
     step_inds = []
     for epoch in range(num_epochs):
         inds = np.arange(len(training.data))
         np.random.shuffle(inds)
-        step_inds.append(inds)
+        batched_inds = []
+        num_steps = int(np.ceil(len(inds) / batch_size))
+        for i in range(num_steps):
+            offset = i * batch_size
+            batched_inds.append(inds[offset:offset+batch_size])
+        step_inds.append(np.asarray(batched_inds))
 
     np.save(output_path.joinpath('step_indices.npy'), np.hstack(step_inds)[:args.param_updates])
 
     step = 0
-    epoch = 0
     # in each optimizer step, look at one transformation from relative_transformations
-    for steps in step_inds:
+    for epoch in range(num_epochs):
         # Run Validation edges at start of epoch. Unlike NNs we have a reasonable starting
         # point that is worth knowing
-        run_validation_edges(validation, ordered_params, systems, epoch)
-        epoch += 1
-        print(f"Epoch: {epoch}")
-        for i in steps:
-            rfe = training.data[i]
-            if getattr(rfe, "complex_path", None):
-                model = systems[rfe.complex_path]
-            else:
-                model = systems[protein_path]
-            # compute a step, measuring total wall-time
+        run_validation_edges(validation, ordered_params, systems, epoch+1, inference=args.inference_only)
```

Comment:
A bit hacky to run all edges as validation in inference mode, but was quick and easy. Perhaps the function should just be renamed?